src/bulkExport/bulkExport.test.ts

```diff
@@ -38,6 +38,19 @@ describe('getBulkExportResults', () => {
         ]);
     });
 
+    test('happy case with tenantId', async () => {
+        AWSMock.mock('S3', 'listObjectsV2', (params: any, callback: Function) => {
+            callback(null, {
```

Comment:
Check here the prefix passed in is as expected?

Reply:
makes sense. I added the check before merging